aiida_vasp/parsers/vasp.py

```diff
@@ -198,22 +231,58 @@ def missing_critical_file():
 
         return self.result(success=True)
 
-    def _set_parsable_quantities(self):
-        """Set the parsable_quantities dictionary based on parsable_items obtained from the FileParsers."""
-
-        import copy
+    def _check_priority_and_add(self):
+        """Add a parser based on priority and availability of the files to be parsed."""
 
         # Gather all parsable items as defined in the file parsers.
+        # We introduce the concept of priority which can be defined in
+        # FILE_PARSER_SETS as the key 'priority' and a number. If a
+        # quantity is present in several different parsers, the parser
+        # with the highest number will persist. The file also has to exist.
+        # It is still fully possible to create custom parser and/or utilize the
+        # alternatives approach.
+        priority = {}
         for filename, value in self._parsers.iteritems():
```

Comment:
please replace `iteritems` with `items` or use `six.iteritems`. Keep in mind, end of life for py2 is 2020.

Reply:
Fixed